buildtools/templatecache.mako.js

```diff
@@ -23,8 +23,6 @@
               content = htmlmin.minify(
                   content,
                   remove_comments=True,
-                  remove_empty_space=True,
-                  remove_all_empty_space=True,
```

Comment:
Your sure that you should remove booth?

Reply:
yes, I tried to remove only remove_all_empty_space but it didn't work

Reply:
???

Reply:
OK, thanks :-)